src/@next/TextArea/TextAreaStyle.ts

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import { borderRadius4 } from '../utilities/borderRadius';
+import { Neutral, Red } from '../utilities/colors';
+import { NotoSans } from '../utilities/fonts';
+
+export const StyledTextAreaContainer = styled.div`
+  position: relative;
+  width: 100%;
+  max-width: 520px;
+`;
+
+export const StyledTextArea = styled.textarea`
```

Comment:
@michac789 Currently, the overflowing text may overlap with the character counter, which is not in line with the design. As the row is set to 3, we want to avoid the text breaking into the 4th line.
![image](https://github.com/glints-dev/glints-aries/assets/12844836/c11c8f0a-ea21-441f-b223-fe3c30987d41)